### PR TITLE
[Docs] Fix typos, improve, update at Conceptual Guides page

### PR DESCRIPTION
**Pull request by standardAI**
# What does this PR do?

Hi! Thank you for this awesome library!
I am new to open source. Firstly, I am reading the documentation. Secondly, I will read the codebase. Thirdly, I plan to make documentation and code contributions. I am open to your guidance.
I would be grateful for your feedback. @stevhliu @yiyixuxu @patrickvonplaten

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/diffusers/blob/main/CONTRIBUTING.md)?
- [X] Did you read our [philosophy doc](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) (important for complex PRs)?
- [ ] Was this discussed/approved via a GitHub issue or the [forum](https://discuss.huggingface.co/)? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/diffusers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by DN6** [comment]
cc'ign @stevhliu

File: docs/source/en/conceptual/contribution.md [hunk 1]
```diff
@@ -28,19 +28,19 @@ the core library.
...
 * 7. Contribute to the [examples](https://github.com/huggingface/diffusers/tree/main/examples).
 * 8. Fix a more difficult issue, marked by the "Good second issue" label, see [here](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+second+issue%22).
 * 9. Add a new pipeline, model, or scheduler, see ["New Pipeline/Model"](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+pipeline%2Fmodel%22) and ["New scheduler"](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+scheduler%22) issues. For this contribution, please have a look at [Design Philosophy](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md).
 
 As said before, **all contributions are valuable to the community**.
 In the following, we will explain each contribution a bit more in detail.
 
-For all contributions 4.-9. you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr)
+For all contributions 4.-9. you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr).
```
Comment (stevhliu) [inline docs/source/en/conceptual/contribution.md:43]:
```suggestion
For all contributions 4 - 9, you will need to open a PR. It is explained in detail how to do so in [Opening a pull request](#how-to-open-a-pr).
```

File: docs/source/en/conceptual/contribution.md [hunk 2]
```diff
@@ -271,7 +271,7 @@ with Diffusers.
 Once an example script works, please make sure to add a comprehensive `README.md` that states how to use the example exactly. This README should include:
 - An example command on how to run the example script as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/dreambooth#running-locally-with-pytorch).
 - A link to some training results (logs, models, ...) that show what the user can expect as shown [here e.g.](https://api.wandb.ai/report/patrickvonplaten/xm6cd5q5).
-- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
+- If you are adding a non-official/research training example, **please don't forget** to add a sentence that you are maintaining this training example which includes your git handle as shown [here e.g.](https://github.com/huggingface/diffusers/tree/main/examples/research_projects/intel_opts#diffusers-examples-with-intel-optimizations).
```
Comment (stevhliu) [inline docs/source/en/conceptual/contribution.md:274]: No need to add `e.g.`

File: docs/source/en/conceptual/contribution.md [hunk 3]
```diff
@@ -297,7 +297,7 @@ if you don't know yet what specific component you would like to add:
 - [Model or pipeline](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+pipeline%2Fmodel%22)
 - [Scheduler](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22New+scheduler%22)
 
-Before adding any of the three components, it is strongly recommended that you give the [Philosophy guide](https://github.com/huggingface/diffusers/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+second+issue%22) a read to better understand the design of any of the three components. Please be aware that
+Before adding any of the three components, it is strongly recommended that you give the [Philosophy guide](https://github.com/huggingface/diffusers/blob/main/PHILOSOPHY.md) a read to better understand the design of any of the three components. Please be aware that
```
Comment (stevhliu) [inline docs/source/en/conceptual/contribution.md:300]: Let's link to the Philosophy guide in the docs

```suggestion
Before adding any of the three components, it is strongly recommended that you give the [Philosophy guide](philosophy) a read to better understand the design of any of the three components. Please be aware that
```

File: docs/source/en/conceptual/evaluation.md [hunk 4]
```diff
@@ -87,7 +87,7 @@ import torch
 seed = 0
 generator = torch.manual_seed(seed)
 
-images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator, output_type="numpy").images
+images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator).images
```
Comment (stevhliu) [inline docs/source/en/conceptual/evaluation.md:90]: Do we need to specify the `output_type` @sayakpaul or is it ok to just leave it as the default `pil` image?

```suggestion
images = sd_pipeline(sample_prompts, num_images_per_prompt=1, generator=generator, output_type="np").images
```

File: docs/source/en/conceptual/philosophy.md [hunk 5]
```diff
@@ -22,22 +22,22 @@ In a nutshell, Diffusers is built to be a natural extension of PyTorch. Therefor
...
-- Diffusers aim at being a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
+- Diffusers aims at being a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
```
Comment (stevhliu) [inline docs/source/en/conceptual/philosophy.md:25]:
```suggestion
- Diffusers aims to be a **light-weight** package and therefore has very few required dependencies, but many soft dependencies that can improve performance (such as `accelerate`, `safetensors`, `onnx`, etc...). We strive to keep the library as lightweight as possible so that it can be added without much concern as a dependency on other packages.
```

File: docs/source/en/conceptual/philosophy.md [hunk 6]
```diff
@@ -22,22 +22,22 @@ In a nutshell, Diffusers is built to be a natural extension of PyTorch. Therefor
...
 - Diffusers prefers simple, self-explainable code over condensed, magic code. This means that short-hand code syntaxes such as lambda functions, and advanced PyTorch operators are often not desired.
 
 ## Simple over easy
 
-As PyTorch states, **explicit is better than implicit** and **simple is better than complex**. This design philosophy is reflected in multiple parts of the library: 
+As The Zen of Python states, **explicit is better than implicit** and **simple is better than complex**. This design philosophy is reflected in multiple parts of the library:
```
Comment (stevhliu) [inline docs/source/en/conceptual/philosophy.md:30]: This is inspired by PyTorch so lets keep that here

File: docs/source/en/conceptual/philosophy.md [hunk 7]
```diff
@@ -83,14 +83,14 @@ The following design principles are followed:
...
 - Models **do not** follow the single-file policy and should make use of smaller model building blocks, such as [`attention.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention.py), [`resnet.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/resnet.py), [`embeddings.py`](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/embeddings.py), etc... **Note**: This is in stark contrast to Transformers' modeling files and shows that models do not really follow the single-file policy.
-- Models intend to expose complexity, just like PyTorch's module does, and give clear error messages.
+- Models intend to expose complexity, just like PyTorch's Module does, and give clear error messages.
```
Comment (stevhliu) [inline docs/source/en/conceptual/philosophy.md:86]: No need to capitalize "module"

File: docs/source/en/conceptual/philosophy.md [hunk 8]
```diff
@@ -83,14 +83,14 @@ The following design principles are followed:
...
 - The model design is a difficult trade-off between keeping code readable and concise and supporting many model checkpoints. For most parts of the modeling code, classes shall be adapted for new model checkpoints, while there are some exceptions where it is preferred to add new classes to make sure the code is kept concise and 
-readable longterm, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
+readable long term, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
```
Comment (stevhliu) [inline docs/source/en/conceptual/philosophy.md:93]:
```suggestion
readable long-term, such as [UNet blocks](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/unet_2d_blocks.py) and [Attention processors](https://github.com/huggingface/diffusers/blob/main/src/diffusers/models/attention_processor.py).
```

File: docs/source/en/using-diffusers/controlling_generation.md [hunk 9]
```diff
@@ -230,15 +207,11 @@ To know more details, check out the [official doc](../api/pipelines/diffedit).
...
 depth maps, and semantic segmentations.
 
-See [here](../api/pipelines/stable_diffusion/adapter) for more information on how to use it.
-
-## Fabric
+## Fabric  <!--Documentation is not yet available, so the link is broken now.-->
 
 [Paper](https://arxiv.org/abs/2307.10159)
 
 [Fabric](../api/pipelines/fabric) is a training-free
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/controlling_generation.md:214]:
```suggestion
[Fabric](https://github.com/huggingface/diffusers/tree/442017ccc877279bcf24fbe92f92d3d0def191b6/examples/community#stable-diffusion-fabric-pipeline) is a training-free
```

File: docs/source/en/using-diffusers/controlling_generation.md [hunk 10]
```diff
@@ -230,15 +207,11 @@ To know more details, check out the [official doc](../api/pipelines/diffedit).
 There are 8 canonical pre-trained adapters trained on different conditionings such as edge detection, sketch,
 depth maps, and semantic segmentations.
 
-See [here](../api/pipelines/stable_diffusion/adapter) for more information on how to use it.
-
-## Fabric
+## Fabric  <!--Documentation is not yet available, so the link is broken now.-->
```
Comment (stevhliu) [inline docs/source/en/using-diffusers/controlling_generation.md:210]:
```suggestion
## Fabric
```

**Review by stevhliu — APPROVED** [review]
Good job, thanks for all the fixes!

File: docs/source/en/conceptual/contribution.md (hunk 2 above)
Comment (standardAI) [reply docs/source/en/conceptual/contribution.md:274]: But the first and second items have "e.g." at their "here" part 😂.

File: docs/source/en/conceptual/philosophy.md (hunk 7 above)
Comment (standardAI) [reply docs/source/en/conceptual/philosophy.md:86]: It is capitalized here: https://huggingface.co/docs/diffusers/main/en/conceptual/philosophy#:~:text=PyTorch%E2%80%99s%20Module%20class

File: docs/source/en/conceptual/contribution.md (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/conceptual/contribution.md:274]: Let's also change those while we're at it then! 😄  

In general, I think adding Latin abbreviations like "e.g" or "i.e" is not as understandable as just writing it out in plain English. But here, we don't need to say "e.g." since earlier in this sentence we already said it was an "example"

File: docs/source/en/conceptual/philosophy.md (hunk 7 above)
Comment (stevhliu) [reply docs/source/en/conceptual/philosophy.md:86]: Let's make it more clear then:

```suggestion
- Models intend to expose complexity, just like PyTorch's `Module` class, and give clear error messages.
```

File: docs/source/en/conceptual/contribution.md (hunk 2 above)
Comment (standardAI) [reply docs/source/en/conceptual/contribution.md]: Done.

File: docs/source/en/conceptual/evaluation.md (hunk 4 above)
Comment (standardAI) [reply docs/source/en/conceptual/evaluation.md:90]: Here, I thought we show images, so `pil`.
